### ISSUE TITLE
python: command not found

### ISSUE THREAD

**Issue by mosoriob** [In-For-Disaster-Analytics/LLMRepository-Docker#7]
The `run.sh` scripts fails

```
+ '[' '!' -d /work/07025/mosorio/ls6/sites-and-stories-nlp-jupyterenv ']'
+ echo 'Installing Conda env'
Installing Conda env
+ python -m ipykernel install --user --name llm --display-name 'Python (llm)'
/tapis/run.sh: line 55: python: command not found
```

## Solution

Use python3

**Comment by mosoriob** [comment]
The logs are: 

```
$ singularity run docker://ghcr.io/in-for-disaster-analytics/llmrepository-docker:sha-6b349fd

c302-005.ls6(272)$ singularity run docker://ghcr.io/in-for-disaster-analytics/llmrepository-docker:sha-6b349fd
INFO:    Using cached SIF image
++ date
+ echo 'TACC: job 1400778 execution at: Sat Dec 16 05:03:02 AM CST 2023'
TACC: job 1400778 execution at: Sat Dec 16 05:03:02 AM CST 2023
+ echo load cuda
load cuda
+ module load cuda/12.0
+ '[' -z '' ']'
+ case "$-" in
+ __lmod_sh_dbg=x
+ '[' -n x ']'
+ set +x
Shell debugging temporarily silenced: export LMOD_SH_DBG_ON=1 for Lmod's output
environment: line 17: /opt/apps/lmod/lmod/libexec/lmod: No such file or directory
Shell debugging restarted
+ unset __lmod_sh_dbg
+ return 0
+ echo 'import torch
print(torch.cuda.is_available())
'
+ python3
False
+ SESSION_FILE=delete_me_to_end_session
+ touch delete_me_to_end_session
+ LOCAL_IPY_PORT=8888
+ LOCAL_PORT=5902
++ hostname -s
+ NODE_HOSTNAME_PREFIX=c302-005
++ hostname -d
+ NODE_HOSTNAME_DOMAIN=ls6.tacc.utexas.edu
++ hostname -f
+ NODE_HOSTNAME_LONG=c302-005.ls6.tacc.utexas.edu
+ echo 'Checking if miniconda3 is installed...'
Checking if miniconda3 is installed...
+ '[' '!' -d /work/07025/mosorio/ls6/miniconda3 ']'
+ echo 'Initializing conda...'
Initializing conda...
+ /work/07025/mosorio/ls6/miniconda3/bin/conda init bash
no change     /work/07025/mosorio/ls6/miniconda3/condabin/conda
no change     /work/07025/mosorio/ls6/miniconda3/bin/conda
no change     /work/07025/mosorio/ls6/miniconda3/bin/conda-env
no change     /work/07025/mosorio/ls6/miniconda3/bin/activate
no change     /work/07025/mosorio/ls6/miniconda3/bin/deactivate
no change     /work/07025/mosorio/ls6/miniconda3/etc/profile.d/conda.sh
no change     /work/07025/mosorio/ls6/miniconda3/etc/fish/conf.d/conda.fish
no change     /work/07025/mosorio/ls6/miniconda3/shell/condabin/Conda.psm1
no change     /work/07025/mosorio/ls6/miniconda3/shell/condabin/conda-hook.ps1
no change     /work/07025/mosorio/ls6/miniconda3/lib/python3.11/site-packages/xontrib/conda.xsh
no change     /work/07025/mosorio/ls6/miniconda3/etc/profile.d/conda.csh
modified      /home1/07025/mosorio/.bashrc

==> For changes to take effect, close and re-open your current shell. <==

+ echo 'Sourcing .bashrc...'
Sourcing .bashrc...
+ source /home1/07025/mosorio/.bashrc
++ '[' -n '' ']'
++ '[' -z 1 -a BATCH '!=' BATCH ']'
++ '[' -z 1 ']'
++ '[' -n '' ']'
+++ /work/07025/mosorio/ls6/miniconda3/bin/conda shell.bash hook
++ __conda_setup='export CONDA_EXE='\''/work/07025/mosorio/ls6/miniconda3/bin/conda'\''
export _CE_M='\'''\''
export _CE_CONDA='\'''\''
export CONDA_PYTHON_EXE='\''/work/07025/mosorio/ls6/miniconda3/bin/python'\''

# Copyright (C) 2012 Anaconda, Inc
# SPDX-License-Identifier: BSD-3-Clause
__conda_exe() (
    "$CONDA_EXE" $_CE_M $_CE_CONDA "$@"
)

__conda_hashr() {
    if [ -n "${ZSH_VERSION:+x}" ]; then
        \rehash
    elif [ -n "${POSH_VERSION:+x}" ]; then
        :  # pass
    else
        \hash -r
    fi
}

__conda_activate() {
    if [ -n "${CONDA_PS1_BACKUP:+x}" ]; then
        # Handle transition from shell activated with conda <= 4.3 to a subsequent activation
        # after conda updated to >= 4.4. See issue #6173.
        PS1="$CONDA_PS1_BACKUP"
        \unset CONDA_PS1_BACKUP
    fi
    \local ask_conda
    ask_conda="$(PS1="${PS1:-}" __conda_exe shell.posix "$@")" || \return
    \eval "$ask_conda"
    __conda_hashr
}

__conda_reactivate() {
    \local ask_conda
    ask_conda="$(PS1="${PS1:-}" __conda_exe shell.posix reactivate)" || \return
    \eval "$ask_conda"
    __conda_hashr
}

conda() {
    \local cmd="${1-__missing__}"
    case "$cmd" in
        activate|deactivate)
            __conda_activate "$@"
            ;;
        install|update|upgrade|remove|uninstall)
            __conda_exe "$@" || \return
            __conda_reactivate
            ;;
        *)
            __conda_exe "$@"
            ;;
    esac
}

if [ -z "${CONDA_SHLVL+x}" ]; then
    \export CONDA_SHLVL=0
    # In dev-mode CONDA_EXE is python.exe and on Windows
    # it is in a different relative location to condabin.
    if [ -n "${_CE_CONDA:+x}" ] && [ -n "${WINDIR+x}" ]; then
        PATH="$(\dirname "$CONDA_EXE")/condabin${PATH:+":${PATH}"}"
    else
        PATH="$(\dirname "$(\dirname "$CONDA_EXE")")/condabin${PATH:+":${PATH}"}"
    fi
    \export PATH

    # We'\''re not allowing PS1 to be unbound. It must at least be set.
    # However, we'\''re not exporting it, which can cause problems when starting a second shell
    # via a first shell (i.e. starting zsh from bash).
    if [ -z "${PS1+x}" ]; then
        PS1=
    fi
fi'
++ '[' 0 -eq 0 ']'
++ eval 'export CONDA_EXE='\''/work/07025/mosorio/ls6/miniconda3/bin/conda'\''
export _CE_M='\'''\''
export _CE_CONDA='\'''\''
export CONDA_PYTHON_EXE='\''/work/07025/mosorio/ls6/miniconda3/bin/python'\''

# Copyright (C) 2012 Anaconda, Inc
# SPDX-License-Identifier: BSD-3-Clause
__conda_exe() (
    "$CONDA_EXE" $_CE_M $_CE_CONDA "$@"
)

__conda_hashr() {
    if [ -n "${ZSH_VERSION:+x}" ]; then
        \rehash
    elif [ -n "${POSH_VERSION:+x}" ]; then
        :  # pass
    else
        \hash -r
    fi
}

__conda_activate() {
    if [ -n "${CONDA_PS1_BACKUP:+x}" ]; then
        # Handle transition from shell activated with conda <= 4.3 to a subsequent activation
        # after conda updated to >= 4.4. See issue #6173.
        PS1="$CONDA_PS1_BACKUP"
        \unset CONDA_PS1_BACKUP
    fi
    \local ask_conda
    ask_conda="$(PS1="${PS1:-}" __conda_exe shell.posix "$@")" || \return
    \eval "$ask_conda"
    __conda_hashr
}

__conda_reactivate() {
    \local ask_conda
    ask_conda="$(PS1="${PS1:-}" __conda_exe shell.posix reactivate)" || \return
    \eval "$ask_conda"
    __conda_hashr
}

conda() {
    \local cmd="${1-__missing__}"
    case "$cmd" in
        activate|deactivate)
            __conda_activate "$@"
            ;;
        install|update|upgrade|remove|uninstall)
            __conda_exe "$@" || \return
            __conda_reactivate
            ;;
        *)
            __conda_exe "$@"
            ;;
    esac
}

if [ -z "${CONDA_SHLVL+x}" ]; then
    \export CONDA_SHLVL=0
    # In dev-mode CONDA_EXE is python.exe and on Windows
    # it is in a different relative location to condabin.
    if [ -n "${_CE_CONDA:+x}" ] && [ -n "${WINDIR+x}" ]; then
        PATH="$(\dirname "$CONDA_EXE")/condabin${PATH:+":${PATH}"}"
    else
        PATH="$(\dirname "$(\dirname "$CONDA_EXE")")/condabin${PATH:+":${PATH}"}"
    fi
    \export PATH

    # We'\''re not allowing PS1 to be unbound. It must at least be set.
    # However, we'\''re not exporting it, which can cause problems when starting a second shell
    # via a first shell (i.e. starting zsh from bash).
    if [ -z "${PS1+x}" ]; then
        PS1=
    fi
fi'
+++ export CONDA_EXE=/work/07025/mosorio/ls6/miniconda3/bin/conda
+++ CONDA_EXE=/work/07025/mosorio/ls6/miniconda3/bin/conda
+++ export _CE_M=
+++ _CE_M=
+++ export _CE_CONDA=
+++ _CE_CONDA=
+++ export CONDA_PYTHON_EXE=/work/07025/mosorio/ls6/miniconda3/bin/python
+++ CONDA_PYTHON_EXE=/work/07025/mosorio/ls6/miniconda3/bin/python
+++ '[' -z x ']'
++ unset __conda_setup
+ conda info
+ local cmd=info
+ case "$cmd" in
+ __conda_exe info
+ /work/07025/mosorio/ls6/miniconda3/bin/conda info

     active environment : None
            shell level : 0
       user config file : /home1/07025/mosorio/.condarc
 populated config files : /home1/07025/mosorio/.condarc
          conda version : 23.10.0
    conda-build version : not installed
         python version : 3.11.5.final.0
       virtual packages : __archspec=1=zen3
                          __glibc=2.35=0
                          __linux=4.18.0=0
                          __unix=0=0
       base environment : /work/07025/mosorio/ls6/miniconda3  (writable)
      conda av data dir : /work/07025/mosorio/ls6/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /work/07025/mosorio/ls6/miniconda3/pkgs
                          /home1/07025/mosorio/.conda/pkgs
       envs directories : /work/07025/mosorio/ls6/miniconda3/envs
                          /home1/07025/mosorio/.conda/envs
               platform : linux-64
             user-agent : conda/23.10.0 requests/2.31.0 CPython/3.11.5 Linux/4.18.0-372.26.1.el8_6.x86_64 ubuntu/22.04.3 glibc/2.35 solver/libmamba conda-libmamba-solver/23.11.1 libmambapy/1.5.3
                UID:GID : 863242:820466
             netrc file : None
           offline mode : False


+ '[' '!' -d /work/07025/mosorio/ls6/sites-and-stories-nlp-jupyterenv ']'
+ echo 'Installing Conda env'
Installing Conda env
+ python -m ipykernel install --user --name llm --display-name 'Python (llm)'
/tapis/run.sh: line 55: python: command not found
```

**Comment by mosoriob** [comment]
Fixed by #8
